internal/tracers/zipkin_tracer.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+
+package tracer
+
+import (
+	"context"
+	"io/ioutil"
+	"strings"
+
+	traceAPI "go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/exporters/trace/zipkin"
+
+	"go.opentelemetry.io/otel/api/global"
+
+	"go.opentelemetry.io/otel/sdk/trace"
+
+	stdLog "log"
+)
+
+// InitTracing initializes a trace provider
+func InitTracing(uri, name string, prob float64) (*trace.Provider, error) {
+	if len(strings.TrimSpace(uri)) == 0 {
+		return nil, nil
```

Comment:
Perhaps return an error value here instead of nil, otherwise we're implying that tracing was initiated?

Reply:
fixed